README.md

```diff
@@ -59,7 +59,7 @@ The directory where your Brewfile is located.
 
 ## Dependencies
 
-None.
+* [elliotweiser.osx-command-line-tools](https://galaxy.ansible.com/elliotweiser/osx-command-line-tools/)
```

Comment:
By defining the dependency in `meta/main.yml`, the dependency role is executed first. 

I believe Apple allows you to run up to VMs (virtualbox or otherwise) as long as it they _run on Mac hardware_. I'll see if I can get my hands on the actual legalese...